faq/faq.md

```diff
@@ -0,0 +1,351 @@
+licenses separately.
+
+If you do not want to include both licenses separately (e.g., for space
+reasons), you can create the file `LICENSES/GPL-3.0.txt` and add the following
+header:
+
+```
+Valid-License-Identifier: GPL-3.0-only
+Valid-License-Identifier: GPL-3.0-or-later
+License-Text:
+
+[license text]
+```
+
+## What are license exceptions and what do I do with them? {#license-exceptions}
+
+## Do I use SPDX-Copyright, Copyright, or ©? {#copyright-symbol}
+
+## Which files are copyrightable? {#what-is-copyrightable}
+
+All files that are original works of authorship are copyrightable. In essence,
+if someone sat down typing their own original thoughts on a keyboard, then that
+person holds copyright over the output. Common examples are source code,
+documentation, audio, and video.
+
+There are some edge cases, however. For instance, the program `print("Hello,
+REUSE!")` probably does not meet the threshold of originality. Similarly, data
+files and configuration files may not meet that threshold either.
+
+## What to do with uncopyrightable files? {#uncopyrightable}
+
+There are two things that you can do with such a file to make sure that its
+copyright and licensing is recorded. The first option is to simply use your
+regular copyright and license header for this file. There is nothing that stops
+you from claiming copyright over your own works.
+
+The alternative is to waive your copyright by using the
+[CC0](https://creativecommons.org/publicdomain/zero/1.0/) license.
+
+It is important to note that you can only do this for your own works. If the
+file was authored by someone else, you must declare their copyright and license
+in the header.
+
+## How do I copy someone else's work? {#copy-work}
+
+If someone else has made their work available for you to use and copy, you can
+incorporate their work into your project. When you put the work in one of the
+files in your project, you should add an `SPDX-Copyright` tag for the copyright
+holder(s) and an `SPDX-License-Identifier` tag for the license(s) under which
+the work was made available.
+
+If the work was licensed differently from your project, you should verify
+whether the licenses are compatible, and add the new licenses to your project.
+
+## Can I copy a work that has no copyright notice or license? {#no-copyright-license}
+
+Before you proceed, always first make sure that you can find the copyright and
+licensing information elsewhere. Some projects only include this information in
+the root directory or in their README file.
+
+If you can find no copyright notice, then that is no problem. You can add the
+copyright notice yourself.
+
+If the work has no license, then that means that you do not have the right to
+copy it. If you believe that this is a mistake and the author clearly meant for
+you to be able to copy this work, you should contact the author and ask them to
+license their work.
+
+## Where else do I put my license information? {#where-else}
+
+README
```

Comment:
Preferably in the SPDX annotation.

Reply:
Agreed. Was also going to include stuff like `setup.py` and other such packaging formats.